pom.xml

```diff
@@ -125,27 +125,25 @@
         </configuration>
       </plugin>
       <plugin>
-	<groupId>org.codehaus.mojo</groupId>
-	<artifactId>animal-sniffer-maven-plugin</artifactId>
-	<version>1.7</version>
```

Comment:
The plugin version is inherited from the parent and overriding it here with an older version is bad and does NOT make it more reproducible. If anything you want to override it with a newer version .. but generally this should be inherited from the parent.,